src/web3/ternoaConnect.tsx

```diff
@@ -133,7 +135,7 @@ export default function TernoaConnect() {
                 // Here we will await the Wallet's response to the pairing proposal
                 const session = await approval();
                 onSessionConnected(session);
-                const nonce = await generateNonce(localStorage.getItem("walletTernoa"));
+                const nonce = await generateNonce(localStorage.getItem("walletTernoa") || '');
```

Comment:
Nous sommes après l'approbation de la connection avec WalletConnect, on doit toujours connaitre l'adresse à cette endroit

Reply:
![image](https://github.com/We-Are-Swissborg/WASB-Front/assets/101713837/b8a6347b-2a1b-4076-9a5f-862c56f747f1)

C'est pour éviter l'erreur comme je l'avais préciser ci-dessus